mopidy/mpd/protocol/stored_playlists.py

```diff
@@ -38,6 +37,17 @@ def listplaylist(context, name):
     return ['file: %s' % t.uri for t in playlist.tracks]
 
 
+def _lookup_playlist(context, name):
+    uri = context.lookup_playlist_uri_from_name(name)
+    playlist = uri is not None and context.core.playlists.lookup(uri).get()
```

Comment:
Would it be cleaner to spread this across normal ifs statements instead of `statement and ...`?